wgpu-core/Cargo.toml

```diff
@@ -49,16 +49,26 @@ package = "wgpu-types"
 version = "0.6"
 
 [target.'cfg(any(target_os = "ios", target_os = "macos"))'.dependencies]
-gfx-backend-metal = { version = "0.6" }
-gfx-backend-vulkan = { version = "0.6", optional = true }
+gfx-backend-metal = { git = "https://github.com/gfx-rs/gfx.git", rev = "c9fcd08956eeddac8f6d85b02794fc5e3d1717f5" }
+gfx-backend-vulkan = { git = "https://github.com/gfx-rs/gfx.git", rev = "c9fcd08956eeddac8f6d85b02794fc5e3d1717f5", optional = true }
 
 [target.'cfg(all(unix, not(target_os = "ios"), not(target_os = "macos")))'.dependencies]
-gfx-backend-vulkan = { version = "0.6.1" }
+gfx-backend-vulkan = { git = "https://github.com/gfx-rs/gfx.git", rev = "c9fcd08956eeddac8f6d85b02794fc5e3d1717f5", features = ["x11"]}
+
+[target.'cfg(unix)'.dependencies]
+# TODO: Make x11 feature optional?
+gfx-backend-gl = { git = "https://github.com/gfx-rs/gfx.git", rev = "c9fcd08956eeddac8f6d85b02794fc5e3d1717f5", features = ["x11"] }
```

Comment:
Should X11 be a Cargo feature for WGPU?

Reply:
No. The idea (so far) is that if the crate that creates a window and passes it to wgpu wants to support x11, it can enable the x11 dependency on gfx-backend-vulkan directly (like wgpu-rs does).